shared.go

```diff
@@ -27,6 +28,9 @@ func LocalesForProjects(client *phraseapp.Client, projectLocales ProjectLocales,
 		if _, ok := projectIdToLocales[key]; !ok {
 			remoteLocales, err := RemoteLocales(client, key)
 			if err != nil {
+				if branch != "" {
```

Comment:
@kirchner I think we should leave a comment here why this case is handled differently. Without context this is a bit hard to understand I think. It might also be better to check for the specific error